src/hooks/useConstitutionLinter.ts

```
@@ -0,0 +1,116 @@
+// =============================================================================
+// Hook: useConstitutionLinter
+// Issue: #3536 - Implement 'Club Constitution Conflict Resolver'
+// Description: Manages the upload of PDF constitutions, triggers the Edge 
```

ESLint

[error] Delete `·`

(prettier/prettier)